objects/rct2ww/scenery_large/rct2.ww.eiffel.json

```diff
@@ -6,9 +6,9 @@
     "sourceGame": "rct2ww",
     "objectType": "scenery_large",
     "properties": {
-        "price": 190,
-        "removalPrice": -150,
-        "cursor": "CURSOR_STATUE_DOWN",
+        "price": 204,
+        "removalPrice": -168,
+        "cursor": "CURSOR_HOUSE_DOWN",
```

Comment:
Not sure how the house cursor is more appropriate for the Eiffel Tower. Is there a rationale?

Reply:
The house cursor appears to be used for any large building. The Colosseum also uses it, for example.